projects/plugins/crm/src/automation/class-automation-workflow.php

```diff
@@ -77,23 +93,39 @@ class Automation_Workflow {
 	 * @since $$next-version$$
 	 * @var int
 	 */
-	public $version = 1;
+	protected $version;
+
+	/**
+	 * A timestamp that reflects when the workflow was created.
+	 *
+	 * @since $$next-version$$
+	 * @var int
+	 */
+	protected $created_at;
+
+	/**
+	 * A timestamp that reflects when the workflow was last updated.
+	 *
+	 * @since $$next-version$$
+	 * @var int
+	 */
+	protected $updated_at;
 
 	/**
 	 * The automation engine.
 	 *
 	 * @since $$next-version$$
 	 * @var Automation_Engine
 	 */
-	private $automation_engine;
+	protected $automation_engine;
 
 	/**
 	 * The automation logger.
 	 *
 	 * @since $$next-version$$
 	 * @var Automation_Logger
 	 */
-	private $logger;
+	protected $logger;
```

Comment:
Technically this change is unnecessary for this PR, but it unifies how we set property visibility.